tests/jetstream_test.ts

```diff
@@ -1254,6 +1254,55 @@ Deno.test("jetstream - deliver start time", async () => {
   await cleanup(ns, nc);
 });
 
+Deno.test("jetstream - deliver last per subject", async () => {
```

Comment:
Not even sure checking against 2.3.3 is enough unless you build from main since the feature is not yet in public release?

Reply:
when I build locally, that works currently reported on local build is 2.3.3-beta or something to that effect.